### PR TITLE
Support React 16 and upgrade dependencies

### PR DESCRIPTION
**Pull request by ssssssssssss**

### CODE REVIEW

**Comment by ssssssssssss** [comment]
@aweary Could you please take a look?

File: __tests__/index.js [hunk 1]
```diff
@@ -17,65 +21,67 @@ const mountProgressiveImage = (renderFn) => {
...
+    const _error = console.error;
+    console.error = jest.fn(() => {});
     expect(() => {
       mount(
         <ProgressiveImage>
           <h1>Uh oh!</h1>
         </ProgressiveImage>
       );
     }).toThrow(`ProgressiveImage requires a function as its only child`);
+    console.error = _error;
```
Comment (ryan-roemer) [inline __tests__/index.js]: If the exception fails, you've permanently mutate `console.error` after this test exits. Should refactor in a try/catch or restore in and `afterEach`

**Comment by ssssssssssss** [comment]
Any chance to push a new release? Thanks.

**Comment by aweary** [comment]
@ssssssssssss I've published 0.3.0!

**Pull request merged**